packages/bentley/grouping-mapping-widget/src/api/FindSimilarApi.ts

```diff
@@ -0,0 +1,50 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+import { KeySet } from "@bentley/presentation-common";
+import {
+  Presentation,
+  SelectionChangesListener,
+} from "@bentley/presentation-frontend";
+import {
+  DEFAULT_PROPERTY_GRID_RULESET,
+  PresentationPropertyDataProvider,
+} from "@bentley/presentation-components";
+import { IModelConnection } from "@bentley/imodeljs-frontend";
+
+export interface PropertyProps {
+  keys: KeySet;
+  imodel?: IModelConnection;
+}
+
+/* This class demonstrates the key APIs needed to access formatted property information
+   suitable to present to end users. */
+export class FindSimilarApi {
```

Comment:
should this really be called FindSImilar? Theres an entire feature in DR with this name